docs/getting-started/overview.rst

```diff
@@ -1,6 +1,106 @@
+is important to use those platforms that will provide the best environment for 
+their execution. Thus, having an application platform that can enable the app 
+to be ran on different operating systems with minimal or no changes provides a 
+significant boon. 
+
+**Open Source** because it is proven to be a great way to enable a larger set of 
+platforms, supported by community contribution.
+
+**Better packaging story** that allows the framework to be distributed 
+as a set of packages that developers can pick and choose from, rather than a single, 
+monolithic platform. .NET Core is the first implementation of .NET Platform that will be 
+distributed via `NuGet <http://www.nuget.org/>`_ package manager. 
+
+**Better application isolation** as one of the scenarios for .NET Core is to 
+enable applications to "take" the needed runtime for their execution and deploy 
+it with the application, not depending on shared components on the targeted 
+machine. This actually plays well with the current trends of developing software 
+and using container technologies like Docker for consolidation. 
+
+.NET Framework
+--------------
+
+The .NET Framework is the premier implementation of the .NET Platform available for 
+Windows server and client developers. It is a very powerful, very mature framework, with 
+a huge class library (known as the **Framework Class Libraries**) that supports 
+a wide variety of applications and solutions on Windows. 
+
+There are additional stacks built on top the .NET Framework that allow developers 
+to build applications ranging from console applications, across rich client (WPF) 
+applications to scalable web applications. 
+
+`Windows Forms <https://msdn.microsoft.com/en-us/library/dd30h2yb(v=vs.110).aspx>`_ 
+and `Windows Presentation Foundation (WPF) <https://msdn.microsoft.com/en-us/library/ms754130(v=vs.110).aspx>`_ 
+are two User Interface (UI) stacks that enable you to build desktop applications for Windows 
+operating system. Windows Forms' strength is in its rich support for common databinding as well as 
+access to Windows' native user interface controls. WPF, on the other hand, allows you to exercise 
+much more control over the look and feel of your application. Both of them allow for building very 
+rich desktop applications that run on Windows, and you should pick the one that is suited for your 
+use case. 
+
+`Windows Communication Foundation (WCF) <https://msdn.microsoft.com/en-us/library/ms731082(v=vs.110).aspx>`_ 
+is a set of libraries that comprise the middleware services stack on .NET Framework. 
+It allows you to create services that can communicate through various supported 
+protocols using various data formats, and that can be hosted in any process 
+you choose. This leads to one of the major features of WCF: your services are 
+not tied to any particular hosting strategy or approach.
+
+ASP.NET is a web framework. Being a very rich framework, it has several distinct pieces 
```

Comment:
How about link the first use of ASP.NET to http://asp.net. It will make the paragraphs in this section seem more symmetrical.